packages/react/src/users/hooks/types/useUserAddress.types.ts

```diff
@@ -0,0 +1,3 @@
+export type UseUserAddressOptions = {
+  enableAutoFetch?: boolean;
```

Comment:
IMO we should add the option `fetchConfig` parameter. (client config)

Same suggestion for `useUserAddresses` hook